src/crawlee/browsers/playwright_browser_plugin.py

```diff
@@ -58,12 +58,12 @@ def browser_type(self) -> Literal['chromium', 'firefox', 'webkit']:
 
     @property
     @override
-    def browser_options(self) -> Mapping[str, Any]:
+    def browser_options(self) -> MutableMapping[str, Any]:
```

Comment:
Making this mutable is almost the same as making the property public. Why does it need to be mutable in the first place?

Reply:
Well, only `page_options` have to be mutable since we add the proxy info there. Therefore I updated both page & browser options. Shall I revert it for the `browser_options`?

Reply:
Couldn't we just make a copy when passing the options to `new_page`? Mutating the page options inside the browser plugin may introduce bugs...